gradle-plugin/src/functionalTest/resources/with-functional-test/build.gradle.kts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2022, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import com.google.protobuf.gradle.protobuf
+import com.google.protobuf.gradle.protoc
+import io.spine.internal.dependency.Protobuf
+
+import org.gradle.api.artifacts.dsl.RepositoryHandler
+import org.gradle.api.plugins.jvm.JvmTestSuite
+
+buildscript {
+    io.spine.internal.gradle.doApplyStandard(repositories)
+}
+
+plugins {
+    java
+    id("com.google.protobuf")
+    id("@PROTODATA_PLUGIN_ID@") version "@PROTODATA_VERSION@"
+}
+
+fun RepositoryHandler.addCouple(baseUrl: String) {
+    maven { url = uri("$baseUrl/releases") }
+    maven { url = uri("$baseUrl/snapshots") }
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+
+    addCouple("https://spine.mycloudrepo.io/public/repositories")
+    addCouple("https://europe-maven.pkg.dev/spine-event-engine")
+}
+
+protoData {
+    renderers("io.spine.protodata.test.NoOpRenderer")
+    plugins("io.spine.protodata.test.TestPlugin")
+}
+
+dependencies {
+    protoData("io.spine.protodata:protodata-test-env:+")
+    Protobuf.libs.forEach { implementation(it) }
+}
+
+protobuf {
+    protoc {
+        artifact = io.spine.internal.dependency.Protobuf.compiler
+    }
+}
+
+@Suppress("UNUSED_VARIABLE") // `test` and `functionalTest`
+testing {
+    suites {
+        val test by getting(JvmTestSuite::class) {
```

Comment:
Please comment on this approach. The unused variables still raise questions, even though the warning is suppressed.